scripts/_util.py

```diff
@@ -1,10 +1,10 @@
 from pathlib import Path
-from typing import Optional
+from typing import Optional, Union
```

Comment:
We're using Python 3.10 and above, so we don't neet `typing.Union`, we can just use the new union syntax `X | Y`.
See https://docs.python.org/3/library/stdtypes.html#types-union